### PR TITLE
[WIP] Start-new-chat screen UI changes

### PR DESCRIPTION
**Pull request by GustavoNunes**
### Summary:
UI Changes for the "Start new chat" screen:
- Android now also have the list with the 3 options in the screen (replaces the 3 options in the chat list action button, this was done in another PR).
- New/changed colors, elements, positioning, etc.

#### Android
- Go to chat list screen
- Tap the action button
- Tap new chat option
#### iOS
- Go to chat list screen
- Tap the **+** button in the toolbar.

#### Notes
The steps to reach the screen have been changed in another PR.

status: wip
Android is done, just need to double-check iOS.

### CODE REVIEW

File: src/status_im/components/styles.cljs [hunk 1]
```diff
@@ -9,7 +9,7 @@
 (def color-gray2 "#8f838c93")
 (def color-gray3 "#00000040")
 (def color-gray4 "#939ba1")
-(def color-gray5 "#d9dae1")
+(def color-gray5 "#d9dae188")
```
Comment (flexsurfer) [inline src/status_im/components/styles.cljs]: why did you add 88 ?

Comment (GustavoNunes) [inline src/status_im/components/styles.cljs]: for the 0.5 opacity as shown in zeplin

File: src/status_im/contacts/styles.cljs [hunk 2]
```diff
@@ -174,7 +212,7 @@
    :alignItems     :center
    :justifyContent :center})
 
-; New contact
+                                        ; New contact
```
Comment (flexsurfer) [inline src/status_im/contacts/styles.cljs]: fell asleep on the space key? :)

File: src/status_im/components/styles.cljs (hunk 1 above)
Comment (flexsurfer) [inline src/status_im/components/styles.cljs]: I think it's better to use opacity on an object not in color because it's important what is under this object and this color also using with 1 opacity

Comment (GustavoNunes) [inline src/status_im/components/styles.cljs]: yeah you're right, changing it!

File: src/status_im/components/styles.cljs [hunk 3]
```diff
@@ -18,18 +18,17 @@
...
 (def text2-color color-gray)
 (def text3-color color-blue)
+(def text4-color color-gray4)
 (def online-color color-blue)
 (def new-messages-count-color color-blue-transparent)
 (def chat-background color-light-gray)
 (def selected-message-color "#E4E9ED")
 (def selected-contact-color color-light-blue2)
-(def separator-color color-gray4)
+(def separator-color color-gray5)
```
Comment (GustavoNunes) [inline src/status_im/components/styles.cljs]: Hmm but isn't this the point of having this? 
That color (color-gray5) is the same defined in Zeplin for almost all separators, in chat list and in contacts list also.

Comment (flexsurfer) [inline src/status_im/components/styles.cljs]: yes, but we should try to refresh UI "locally" without affecting the rest of the UI, i've changed color in 0.9.4 from "#0000001f" to color-gray4 but returned back in my last PR, because login screen UI looks bad

Comment (flexsurfer) [inline src/status_im/components/styles.cljs]: but i think there are not many places where it is used, so we can use separator-color in new UI and direct colors in old UI instead of separator-color

Comment (flexsurfer) [inline src/status_im/components/styles.cljs]: and also separator color can be with alpha channel imo, so we will have color-gray5 color without alpha and separator-color same color but with alpha

**Comment by flexsurfer** [comment]
i think we need to create new screen for Start-new-chat functionality, because contact-list using for view/edit group contacts, and in the new interface it is very different in UI and functionality
